assets/stylesheets/_components.scss

```diff
@@ -174,8 +174,7 @@
 @import 'my-sites/exporter/style';
 @import 'my-sites/invites/invite-form-header/style';
 @import 'my-sites/invites/invite-header/style';
-@import 'my-sites/invites/invite-accept/logged-in/style';
-@import 'my-sites/invites/invite-accept/logged-out/style';
+@import 'my-sites/invites/invite-accept-logged-in/style';
```

Comment:
Do we have to change the name to `invite-accept-logged-in`? Or could we just move the component up a directory to `my-sites/invites/logged-in`?